docs/cli-usage.md

```diff
@@ -0,0 +1,96 @@
+## Using Kubernetes from the command line
```

Comment:
Something else we discussed somewhere was descriptions: `--description="This is my foo server for bonking bobbles."` should add a description annotation. We could display descriptions in the UI.

I'll pull this discussion into a cli/config roadmap (#2235).